mesonbuild/modules/pkgconfig.py

```diff
@@ -272,6 +283,11 @@ def generate_libs_flags(libs):
                 for l in libs:
                     if isinstance(l, str):
                         yield l
+                    elif uninstalled:
+                        path = state.backend.get_target_filename_abs(l)
+                        if not path in Lflags:
```

Comment:
__[Flake8]__ <sub>posted by Sider</sub>

[E713] test for membership should be 'not in'